ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -1508,6 +1508,8 @@ DX2ZC:
   - cooling_thermal_power_capacity
   - cooling_percentage_command
   - compressor_speed_percentage_command
+  - compressor_speed_percentage_command_1
```

Comment:
@cstirdivant thoughts on this? does it make sense to have the enums and non enums in the same type?